yelp_package/dockerfiles/lucid/Dockerfile

```diff
@@ -20,6 +20,9 @@ MAINTAINER Kyle Anderson <kwa@yelp.com>
 # Which at this time is in this package
 RUN apt-get update && apt-get -y install dpkg-dev python-tox python-setuptools python-dev libssl-dev libffi-dev debhelper dh-virtualenv python-yaml libyaml-dev python-pytest pyflakes python2.7 python2.7-dev help2man zsh git
 
+RUN pip install -U pip
+RUN pip install -U virtualenv
```

Comment:
Yep, although we weren't really pinning before because we are installing these from apt anyway, so just relying on the docker cache to keep them somewhat the same.

I'd say that if the build becomes stable we can change these lines to `pip install pip==x.y.z` etc